backend/controllers/search/prisonerSearch.ts

```diff
@@ -116,27 +112,17 @@ export default ({
         keywords,
         selectedAlerts,
         location: location || currentUserCaseLoad,
-        feature,
         username,
       })
 
-      let iepBookingIdMap = new Map<number, IepSummaryForBookingId>()
-      if (prisoners.length) {
-        const bookingIds = prisoners.map((prisoner) => prisoner.bookingId)
-        const iepData = await incentivesApi.getIepSummaryForBookingIds(localContext, bookingIds)
-        iepBookingIdMap = toMap('bookingId', iepData)
-      }
-
       const locationOptions =
         locations && locations.map((option) => ({ value: option.locationPrefix, text: option.description }))
 
       const results =
         prisoners &&
         prisoners.map((prisoner) => ({
           ...prisoner,
-          iepLevel:
-            (iepBookingIdMap.has(prisoner.bookingId) && iepBookingIdMap.get(prisoner.bookingId).iepLevel) ||
-            'Not entered',
+          iepLevel: prisoner.currentIncentive?.level.description ?? 'Not entered',
```

Comment:
is this right? Aren't you missing a `?` after `level`?  Surprised that you don't get a warning on this?